pkg/cluster/operation/destroy.go

```diff
@@ -76,6 +76,15 @@ func Destroy(
 		}
 	}
 
+	// Delete systemd directory when use systemd user mode
+	if gOpts.SystemdMode == spec.UserMode {
+		for host := range instCount {
+			if err := DeleteSystemdDir(ctx, host, gOpts); err != nil {
```

Comment:
don't delete it, user may have some custom service

Reply:
get